frontend/components/charts/SignedExtrinsicsChart.vue

```diff
@@ -0,0 +1,268 @@
+<template>
+  <div>
+    <div v-if="loading" class="text-center py-4">
+      <Loading />
+    </div>
+    <div v-else>
+      <ChartFilter :buttons="filterButtons" :active-button="activeButton" />
+      <BarChart
+        id="line"
+        :chart-data="chartData"
+        :options="chartOptions"
+        :height="200"
+      />
+    </div>
+  </div>
+</template>
+<script>
+import BarChart from '@/components/charts/BarChart.js'
+import Loading from '@/components/Loading.vue'
+import ChartFilter from '@/components/ChartFilter.vue'
+import signedExtrinsic from '@/mixins/signedExtrinsic.js'
+
+export default {
+  components: {
+    BarChart,
+    Loading,
+    ChartFilter,
+  },
+  mixins: [signedExtrinsic],
+  data() {
+    return {
+      filterButtons: [
```

Comment:
You should reuse this code between charts. Think about factory methods pattern

Reply:
done